python/docs/source/reference/timestream/aggregation.md

```diff
@@ -9,7 +9,7 @@ It is important to remember that aggregations are partitioned by entity and wind
 ## Aggregation Methods
 
 ```{eval-rst}
-.. currentmodule:: kaskada
+.. currentmodule:: kaskada.timestream
```

Comment:
I'm not sure I like exposing this as a separate module -- `kd.timestream` is pretty long. On the other hand, I could be convinced that the user generally doesn't interact with that module directly. But, this was why it was previously `_timestream` -- there doesn't seem to be a reason to expose this as a public module. Why not keep it `_timestream`?

Reply:
I don't think I can make a module that starts with an underscore.  I tried that at first but I wasn't having much luck getting it working.

Reply:
looking at docs, seems like I should be able to start a module with an underscore. Maybe something else was the issue at that point. I can try to make it private again.

Reply:
resolved.

Reply:
I was also concerned about the longer name length, but users shouldn't ever use the module directly.